secure/storage/src/tests/suite.rs

```diff
@@ -185,43 +184,6 @@ fn test_create_and_get_non_existent_version(storage: &mut dyn Storage) {
     );
 }
 
-/// This test creates a new key pair, performs a key rotation and ensures that the keys
-/// are correctly rotated in storage.
-fn test_create_rotate_and_check_key_pair(storage: &mut dyn Storage) {
-    // Create new key pair, fetch both public and private keys and verify relationship
-    let public_key = storage
-        .create_key(CRYPTO_NAME, &Policy::public())
-        .expect("Failed to create a test Ed25519 key pair!");
-    let private_key = storage
-        .export_private_key(CRYPTO_NAME)
-        .expect("Failed to get the private key for a key pair that should exist!");
-    assert_eq!(private_key.public_key(), public_key);
```

Comment:
nit: if we're going to remove this test, can we add these relationship checks to the test below? i.e., ensure that the public and private keys for a specific key have a relationship? "assert_eq!(private_key.public_key(), public_key);"